src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -62,10 +63,10 @@ public ErrorResponse handleResourceExists(ResourceExistsException e) {
         return produceErrorResponse(e.getMessage(), HttpStatus.CONFLICT);
     }
 
-    @ExceptionHandler(OsiamBackendFailureException.class)
+    @ExceptionHandler({OsiamBackendFailureException.class, ConnectionInitializationException.class})
```

Comment:
But what about the [revoke of the access token](https://github.com/wallner/resource-server/blob/meaningful_log_message/src/main/java/org/osiam/security/authorization/AccessTokenValidationService.java#L116)? It could be also throwing a `ConnectionInitializationException` when the `auth-server` is not available...